src/main/java/com/conveyal/r5/analyst/TravelTimeComputer.java

```diff
@@ -42,18 +42,22 @@ public TravelTimeComputer(AnalysisTask request, TransportNetwork network, GridCa
         this.network = network;
         this.gridCache = gridCache;
     }
+    public TravelTimeComputer(AnalysisTask request, TransportNetwork network) {
+        this.request = request;
+        this.network = network;
+        this.gridCache = null;
+    }
 
     // We should try to decouple the internal representation of the results from how they're serialized to an API.
     public OneOriginResult computeTravelTimes() throws IOException {
-
         // The mode of travel that will be used to reach transit stations from the origin point.
         StreetMode accessMode = LegMode.getDominantStreetMode(request.accessModes);
         // The mode of travel that will be used to reach destinations from transit stations.
         StreetMode egressMode = LegMode.getDominantStreetMode(request.egressModes);
         // The mode of travel that would be used to reach the destination directly without using transit.
         StreetMode directMode = LegMode.getDominantStreetMode(request.directModes);
 
-        // The set of destinations in the one-to-many travel time calculations, not yet linked to the street network.
+        // The set of destinations in the one-to-many travel time calculations, already linked to the street network.
```

Comment:
The comment has been changed, but looking at our code, the pointset is indeed not linked to the street network at this line. We might just not have a shared base of terminology to communicate here. Our definition of linking is associating a point with the closest edge in one specific network. Here we are just fetching the pointset but not yet ensuring that it is linked.